lnwallet/channel.go

```diff
@@ -811,6 +812,19 @@ func (lc *LightningChannel) diskHtlcToPayDesc(feeRate chainfee.SatPerKWeight,
 		}
 	}
 
+	// Reconstruct the proper local/remote output indexes from the HTLC's
+	// persisted output index depending on whose commitment we are
+	// generating.
+	var (
+		localOutputIndex  int32
+		remoteOutputIndex int32
+	)
+	if isLocal {
+		localOutputIndex = htlc.OutputIndex
```

Comment:
Do we actually need these values _after_ signing? Assuming that we never need to re-write a commitment to disk that is.

Reply:
i couldn't find anywhere where we use them after signing, but i think it's safer to reconstruct the in-memory state as accurately as possible to reduce the chance of introducing new bugs.